python/django_bolt/api.py

```
@@ -1848,23 +1867,23 @@ def execute_trivial_async_sync(
                                     coro.close()
                                     raise RuntimeError("Handler awaited unexpectedly in sync dispatch")
                                 if isinstance(result, dict):
-                                    return (default_status, _meta_json, _BODY_BYTES, _encode(result))
+                                    return _encode(result)
                                 if isinstance(result, list):
                                     result = _convert_serializers(result)
-                                    return (default_status, _meta_json, _BODY_BYTES, _encode(result))
+                                    return _encode(result)
                                 return serialize_response_sync(result, meta)
                     else:
                         if _skip_convert:
```

Ruff (0.15.21)

[warning] Use `elif` instead of `else` then `if`, to reduce indentation

Convert to `elif`

(PLR5501)